ghcup

```diff
@@ -577,12 +595,37 @@ get_distro_ver() {
         # shellcheck disable=SC1091
         . /etc/lsb-release
         printf "%s" "$DISTRIB_RELEASE"
+    elif [ -f /etc/redhat-release ]; then
+        case "$(cat /etc/redhat-release)" in
+        # NB: Older CentOS releases didn't have a /etc/centos-release file
+        "CentOS release "*|"Fedora release "*)
+            printf "%s" "$(awk 'NR==1 { split($3, a, "."); print a[1] }' /etc/redhat-release)"
```

Comment:
Are we sure this file cannot have comments or blank lines?

Reply:
I'm very confident; also this is mostly for legacy distro releases anyway (and the ones I've looked at had this uniform single-line `"${NAME} release ${RELEASE}"`-pattern); all recent Linux distros have switched to the `/etc/os-release` format anyway